src/displayBackend/wayland/WaylandZCopy.cpp

```diff
@@ -317,4 +317,111 @@ void WaylandKms::release()
 	}
 }
 
+/*******************************************************************************
+ * Linux dmabuf
+ ******************************************************************************/
+
+WaylandLinuxDmabuf::WaylandLinuxDmabuf(wl_registry* registry,
```

Comment:
registry is nullptr , is it correct ?
what happens if registry is destroyed outside the WaylandLinuxDmabuf ?

Reply:
1. registry is guaranteed by wayland to be non-nullptr otherwise this method won't be called
2. registry must not be destroyed inside, this is going to be a bug otherwise